day_7/10_speckled_frogs.rb

```diff
@@ -0,0 +1,20 @@
+def frogs(y)
+  while y > 1
+   puts "#{y} speckled frogs sat on a log"
+   puts "eating some most delicious bugs."
+   puts "One jumped in the pool where its nice and cool,"
+   puts "then there were #{y - 1} speckled frogs."
+   y -= 1
+  end
+   puts "#{y} speckled frogs sat on a log"
+   puts "eating some most delicious bugs."
+   puts "One jumped in the pool where its nice and cool,"
+   puts "then there were no more speckled frog!"
```

Comment:
Very clean, nice use of iteration. Refactor opportunity to remove repeating lines. Nice job working through an extension.